### PR TITLE
docs: add linking steps to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by nickofthyme**
## Summary

Updated docs to share how to link the library to kibana and other applications.

### Checklist

(https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#cross-browser-compatibility)
- [x] Proper documentation or storybook story was added for features that require explanation or tutorials
- [x] Each commit follows the [convention](https://github.com/elastic/elastic-charts/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -49,3 +49,40 @@ yarn backport
...
+
+> If running repeated runs and you notice changes not being updated after a new install you _may_ need to increment the `@elastic/charts` version for kibana/npm/??? to pick up the changes. This should NOT be commited.
+
+#### Uninstalling
+1. Reset changes to `package.json`.
+1. Run `yarn kbn bootstrap` †. Or `yarn install --force`
+
+### Symlink the library and build in watch mode
+
+The second is a little trickier but my preferred option, especially for fixing bugs
```
Comment (markov00) [inline CONTRIBUTING.md]: I will remove `my preferred option` in favour of: 
`The second is a little trickier but the preferred option for fixing bugs`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -49,3 +49,40 @@ yarn backport
 This will automatically generate a pr to merge into the target branch. Once the pr is merged the ci will trigger the release of the package version according to the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0-beta.2/) syntax. This is subject to [restrictions](https://semantic-release.gitbook.io/semantic-release/usage/workflow-configuration#pushing-to-a-maintenance-branch) from existing branches, tags and version.
 
 > Note: The version will __ONLY__ be published from branches following the [maintenance branch convention](https://github.com/semantic-release/semantic-release/blob/0785a844fa8ac1320383452ce531898be3b01f92/docs/recipes/maintenance-releases.md#publishing-maintenance-releases)
+
+## Linking to Application
+
+There are two ways to "link" a local version of `@elastic/charts` to your application. The following examples have steps that are kibana-specific denoted with †.
```
Comment (markov00) [inline CONTRIBUTING.md]: usually `†` denotes something that is obsolete, extinct, maybe it's better to change it with something else like `☈` or `☞`

Comment (nickofthyme) [reply CONTRIBUTING.md]: really, it just means a footnote in US like * or ‡

**Pull request merged**